packages/components/src/template-registry.ts

```diff
@@ -23,9 +24,12 @@ import type HdsTagComponent from './components/hds/tag';
 import type HdsToastComponent from './components/hds/toast';
 import type HdsTextCodeComponent from './components/hds/text/code';
 import type HdsYieldComponent from './components/hds/yield';
+
 import type HdsLinkToModelsHelper from './helpers/hds-link-to-models';
 import type HdsLinkToQueryHelper from './helpers/hds-link-to-query';
 
+import type HdsClipboardModifier from './modifiers/hds-clipboard';
```

Comment:
see: https://hashicorp.slack.com/archives/C06NMSYDZPA/p1715876987605319?thread_ts=1715872354.358549&cid=C06NMSYDZPA and Alexey's answer
```suggestion
import type HdsClipboardModifier from './modifiers/hds-clipboard.ts';
```